spec/support/shared_examples/participant_actions_resume_support.rb

```diff
@@ -19,3 +19,79 @@
     expect { described_class.call(params: given_params) }.to raise_error(ActiveRecord::RecordInvalid)
   end
 end
+
+RSpec.shared_examples "JSON Participant Resume endpoint" do |serialiser_type|
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  it "changes the training status of a participant to active" do
```

Comment:
Does this test actually check that the training status of a participant changes to active? It looks like it just checks that the response is 200 and is a participant